src/docfx/lib/cache/JsonDiskCache.cs

```diff
@@ -135,14 +135,17 @@ private Lazy<Task<TError>> UpdateDelegate(TKey key, Func<TKey, Task<(TError, IEn
             });
         }
 
-        private static double NextEvenDistribution(double value)
+        private static DateTime GetRandomUpdatedAt()
         {
-            return (value / 2) + (t_random.Value.NextDouble() * value / 2);
+            return DateTime.UtcNow.AddMilliseconds(1000.0 * t_random.Value.NextDouble());
         }
 
-        private static bool HasExpired(TValue value)
+        private bool HasExpired(TValue value)
         {
-            return value.Expiry != null && value.Expiry < DateTime.UtcNow;
+            var updatedAt = value.UpdatedAt ?? ((DateTime)(value.UpdatedAt = GetRandomUpdatedAt()));
+            var expiry = (0.5 + (updatedAt.Millisecond / 1000.0)) * _expirationInSeconds;
```

Comment:
should be 2000 ?

Reply:
I see, the original was 0.5~1, I thought it was 0.5~1.5